charts/px-central/templates/px-monitor/grafana/grafana.yaml

```diff
@@ -192,11 +197,14 @@ spec:
         - name: pxcentral-grafana
           image: {{ printf "%s/%s/%s:%s" .Values.images.grafanaImage.registry .Values.images.grafanaImage.repo .Values.images.grafanaImage.imageName .Values.images.grafanaImage.tag }}
           imagePullPolicy: {{ .Values.images.pullPolicy }}
-          {{- if .Values.caCertsSecretName }}
+          {{- if or .Values.caCertsSecretName .Values.proxy.httpProxy.noProxy }}
```

Comment:
what is this change for?

Reply:
This is required to add `env:` field or not. So we check for the existence of any one and add `env:`